### PR TITLE
fix(postcards): don't require from

### PR DESCRIPTION
**Pull request by robinjoseph08**
### What

Make `from` not required for postcards

### Why

It isn't required so I'm not sure why it is to begin with...

@mgartner

### CODE REVIEW

File: src/test/java/com/lob/client/test/PostcardTest.java [hunk 1]
```diff
@@ -126,6 +126,29 @@ public void testCreatePostcardInline() throws Exception {
     }
 
     @Test
+    public void testCreatePostcardNoFrom() throws Exception {
+        final PostcardRequest request = PostcardRequest.builder()
+            .to(
+                AddressRequest.builder()
+                    .name("Lob0")
```
Comment (brianseitel) [inline src/test/java/com/lob/client/test/PostcardTest.java]: Lob0?

**Comment by brianseitel** [comment]
One comment. LGTM.

File: src/test/java/com/lob/client/test/PostcardTest.java (hunk 1 above)
Comment (robinjoseph08) [inline src/test/java/com/lob/client/test/PostcardTest.java]: ¯\\\_(ツ)_/¯ I copied it from the above test. I can change it if you want though.

Comment (brianseitel) [inline src/test/java/com/lob/client/test/PostcardTest.java]: :+1:

**Comment by elnaz** [comment]
LGTM

**Pull request merged**